test/plugins/slo/slo_test.go

```diff
@@ -4,12 +4,15 @@ import (
 	"context"
 	"time"
 
+	"github.com/gogo/status"
```

Comment:
Keep an eye out for this, since the IDE will often autocomplete this package (or grpc.go4.org) instead of `google.golang.org/grpc/status`, which is the correct one for this project. Make sure to run `go mod tidy` after and check that it is no longer being imported.

Reply:
that's a weird one, but it is fixed now